components/CandidateTestView/TextEditor/TextEditorContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {MutableRefObject, useRef, useState} from "react";
+import Editor from "@monaco-editor/react";
+import {SupportedLanguages, TextEditorOptions, TextEditorThemes} from "../../../Models/TextEditorOptions";
+import scss from "../TextEditor/TextEditorContainer.module.scss";
+import Link from "next/link";
+import {CandidateTestModel} from "../../../Models/CandidateTestModel";
+
+
+type EditorContentGetter = () => string;
+export function getEditorProps(test: CandidateTestModel): TextEditorContainerProps {
+    type SupportedLanguages = 'javascript';
+    const javascript: SupportedLanguages = "javascript";
```

Comment:
we can probably delete this and just use the string "javascript"

Reply:
I tried that initially, editor didn't like it?

Reply:
I think editor has been changed now so that you don't need to pass it in at all.

As I say, seems like you are maybe working of an old branch now and need to merge master back into your branch?